torch/_inductor/codegen/cpp_utils.py

```diff
@@ -397,10 +397,12 @@ class LocalBufferContext:
     def __init__(self, kernel_args: KernelArgs):
         self.kernel_args = kernel_args
         self.exit_stack = contextlib.ExitStack()
-        # Map Local Buffer name to Local Buffer
+        # map local buffer name to local buffer
         self.local_buffers: Dict[str, ir.Buffer] = {}
-        # Map Local Buffer name to Global Buffer
-        self.local_to_global: Dict[str, ir.Buffer] = {}
+        # map global buffer name to global buffer
+        self.global_buffers: Dict[str, ir.Buffer] = {}
```

Comment:
This is tracked by `V.graph`. Do we still need it here?

Reply:
I guess we may need it, for the case of GEMM Template, the global buffer of GEMM out may not be kept in `V.graph`.